dapp/src/components/Footer.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+import {
+  Box,
+  useMultiStyleConfig,
+  List,
+  ListItem,
+  LinkProps,
+  Link,
+  Button,
+  Icon,
+} from "@chakra-ui/react"
+import { EXTERNAL_HREF } from "#/constants"
+import { AcreSignIcon, ArrowUpRight } from "#/assets/icons"
+
+const NAVIGATION: Pick<LinkProps, "href" | "isExternal" | "children">[] = [
+  {
+    children: "Acre.fi",
+    href: EXTERNAL_HREF.WEBSITE,
+    isExternal: true,
+  },
+  {
+    children: "Docs",
+    href: EXTERNAL_HREF.DOCS,
+    isExternal: true,
+  },
+  {
+    children: "FAQ",
+    href: EXTERNAL_HREF.FAQ,
+    isExternal: true,
+  },
+  {
+    children: "Blog",
+    href: EXTERNAL_HREF.BLOG,
+    isExternal: true,
+  },
+  {
+    children: "Discord",
+    href: EXTERNAL_HREF.DISCORD,
+    isExternal: true,
+  },
+  {
+    children: "X",
+    href: EXTERNAL_HREF.X,
+    isExternal: true,
+  },
+]
+
+const DOCUMENTS: Pick<LinkProps, "href" | "isExternal" | "children">[] = [
```

Comment:
Do we need to split these links to `DOCUMENTS` and `NAVIGATION`? They are rendered in the same list so it seems unnecessary.

Reply:
Those are rendered as two separate lists: one aligned to left and the second one - to the right.

Reply:
Ahh I see. I misunderstood this: `[NAVIGATION, DOCUMENTS]`.

Reply:
Refactored due to https://github.com/thesis/acre/pull/777#discussion_r1814940416: f77df1cad2713ea2435e3b4a3b3353969d1225f5